[2025-02-05T06:41:02.217+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:41:02.271+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: task_group_example.end_task scheduled__2024-02-25T00:00:00+00:00 [queued]>
[2025-02-05T06:41:02.294+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: task_group_example.end_task scheduled__2024-02-25T00:00:00+00:00 [queued]>
[2025-02-05T06:41:02.295+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-02-05T06:41:02.314+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): end_task> on 2024-02-25 00:00:00+00:00
[2025-02-05T06:41:02.320+0000] {standard_task_runner.py:57} INFO - Started process 2807 to run task
[2025-02-05T06:41:02.323+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'task_group_example', 'end_task', 'scheduled__2024-02-25T00:00:00+00:00', '--job-id', '302', '--raw', '--subdir', 'DAGS_FOLDER/task_group_dag.py', '--cfg-path', '/tmp/tmp55ukcc6b']
[2025-02-05T06:41:02.324+0000] {standard_task_runner.py:85} INFO - Job 302: Subtask end_task
[2025-02-05T06:41:02.336+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:41:02.351+0000] {task_command.py:415} INFO - Running <TaskInstance: task_group_example.end_task scheduled__2024-02-25T00:00:00+00:00 [running]> on host 3849bfb82152
[2025-02-05T06:41:02.355+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:41:02.381+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='task_group_example' AIRFLOW_CTX_TASK_ID='end_task' AIRFLOW_CTX_EXECUTION_DATE='2024-02-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-25T00:00:00+00:00'
[2025-02-05T06:41:02.382+0000] {logging_mixin.py:151} INFO - 执行任务: 结束任务
[2025-02-05T06:41:02.382+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-05T06:41:02.384+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=task_group_example, task_id=end_task, execution_date=20240225T000000, start_date=20250205T064102, end_date=20250205T064102
[2025-02-05T06:41:02.425+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-05T06:41:02.431+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
